packages/desktop-client/src/components/reports/reports/SpendingCard.tsx

```diff
@@ -170,20 +156,14 @@ export function SpendingCard({
             </View>
           )}
         </View>
-        {!showCompareTo || isDateValid.toString() === 'Invalid Date' ? (
-          <View style={{ padding: 5 }}>
-            <p style={{ margin: 0, textAlign: 'center' }}>
-              <Trans>Additional data required to generate graph</Trans>
-            </p>
-          </View>
-        ) : data ? (
```

Comment:
A bit of a change here.. I think it would be better to always show a graph. Even if it doesn't have much data. Instead of showing this label.

Don't think this is a good idea? Lets discuss!

Reply:
I'm also considering it removing it from the inner report page if there are no objections.

Reply:
Yea, I'm fine with both. They were originally put on there in the first version where date ranges were much more constricted. I don't think either warning message is necessary anymore.

Reply:
Removed: https://github.com/actualbudget/actual/pull/3432/commits/81b01358533c263fc53d2613d02c5a5fec26724c